jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BuildImageTask.java

```diff
@@ -105,9 +106,9 @@ public void buildImage() throws InvalidImageReferenceException {
             .setTargetImageCredentialHelperName(jibExtension.getTo().getCredHelper())
             .setKnownTargetRegistryCredentials(knownTargetRegistryCredentials)
             .setMainClass(mainClass)
-            .setJavaArguments(jibExtension.getContainer().getArgs())
-            .setJvmFlags(jibExtension.getContainer().getJvmFlags())
-            .setTargetFormat(jibExtension.getContainer().getFormat())
+            .setJavaArguments(jibExtension.getArgs())
```

Comment:
This could be left as `jibExtension.getContainer().getArgs()` since `jibExtension.getArgs()` (and the like) will be removed.

Reply:
I changed `jibExtension.getArgs()` to return `container.getArgs()`, and my plan was to tag `jibExtension.getArgs()` with `@Internal` instead of removing it in the future. Sort of like how `getTargetImage()` and `getBaseImage()` are handled.

Reply:
Ah okay that sounds good.